.rubocop_common.yml

```diff
@@ -0,0 +1,63 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-minitest
+
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  TargetRubyVersion: 2.5
+  TargetRailsVersion: 5.2
```

Comment:
Also pull those versions out from the include file(s) as plugins can be targeting different versions.